README.rst

```diff
@@ -0,0 +1,22 @@
+esi-ui
+======
+
+OpenStack Horizon plugin that supports ESI
+
+Installation
+------------
+
+Clone the repository::
+
+        git clone https://github.com/CCI-MOC/esi-ui.git
+
+Copy ``enabled`` file::
+
+        cp -r <repo path>/esi_ui/enabled <horizon path>/openstack_dashboard/local
+
+Install the package into the environment you use to run your server::
+    
+        python setup.py sdist
+        pip install dist/<package>.tar.gz
+
+Finally, restart your server
```

Comment:
I also had to run `collectstatic` and `compress` again; maybe that's just the way my horizon is configured?

Reply:
It all depends on how the user installs horizon. For example, since I am using tox, I would have to edit the horizon/tox.ini file to include the esi-ui package as a dependency.

Reply:
That sounds like a completely different method of installation; if so, the documentation should mention both possible series of steps.